packages/oscal-react-library/src/components/OSCALProperties.tsx

```diff
@@ -101,11 +120,7 @@ const OSCALProperties = (props: OSCALPropertiesProps): ReactElement => {
           </StyledTableHead>
           <TableBody>
             {properties
-              ?.filter((property: any) =>
-                namespace === NIST_DEFAULT_NAMESPACE
-                  ? isNistNamespace(property?.ns)
-                  : property?.ns === namespace
-              )
+              ?.filter((property: any) => namespaceOf(property?.ns) === namespace)
```

Comment:
I do not want any `: any`s as part of this -- we know these are `Property`s and we should make sure that is able to inferred in all these places. If it can't we're doing something wrong somewhere else.

There are a lot of these so I am just going to comment it here instead of everywhere.